boards/shields/weact_ov2640_cam_module/Kconfig.shield

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2024 Charles Dias
+# SPDX-License-Identifier: Apache-2.0
+
+config SHIELD_WEACT_OV2640_CAM_MODULE
+	def_bool $(shields_list_contains,weact_ov2640_cam_module)
```

Comment:
I did not know Zephyr had custom [Kconfig](https://docs.zephyrproject.org/latest/build/kconfig/preprocessor-functions.html) macros!

Reply:
Zephyr is full of surprises! :)